docs/analytics/apis/user-profile-api.md

```diff
@@ -494,6 +494,10 @@ Find the `prediction_id` by navigating to the prediction in the Audiences web ap
 
 recommend.amplitude.com/0000/predictions/**0x10x**
 
+!!!note "Multiple Prediction IDs"
+
+    To fetch multiple prediction_id, separate predictionIDs by comma(,). For example: prediction_id=id1,id2. The response with multiple response will be in `propensities` field instead of `propensity` field
```

Comment:
```suggestion
    To fetch multiple prediction_id, separate predictionIDs by comma(,). For example: `prediction_id=id1,id2`. Responses for multiple prediction IDs are in the `propensities` field.
```
@zc632995195 Do we have a common way to format `prediction_id`? Or is it different in a request (with an underscore) vs what the field is called predictionID?

Reply:
Good point.  The `predictionID` is misleading there (replace with `prediction_id` --> that has to be exact name in the params.